modules/auxiliary/admin/http/allegro_rompager_auth_bypass.rb

```diff
@@ -25,8 +25,8 @@ def initialize(info = {})
               ],
               'References' => [
                   ['CVE', '2014-9222'],
-                  ['URL', 'http://mis.fortunecook.ie'],
-                  ['URL', 'http://mis.fortunecook.ie/misfortune-cookie-suspected-vulnerable.pdf'], # list of likely vulnerable devices
+                  ['URL', 'https://web.archive.org/web/20191006135858/http://mis.fortunecook.ie/'],
+                  ['URL', 'https://web.archive.org/web/20190207102911/http://mis.fortunecook.ie/misfortune-cookie-suspected-vulnerable.pd'], # list of likely vulnerable devices
                   ['URL', 'http://mis.fortunecook.ie/too-many-cooks-exploiting-tr069_tal-oppenheim_31c3.pdf'] # 31C3 presentation with POC
```

Comment:
Presumably this URL should also be updated to use `archive.org`.

The two `mis.fortunecook.ie` URLs above were updated to use `archive.org`. `mis.fortunecook.ie` resolves but does not have ports 80 or 443 open.

```suggestion
                  ['URL', 'https://web.archive.org/web/20190623150837/http://mis.fortunecook.ie/too-many-cooks-exploiting-tr069_tal-oppenheim_31c3.pdf'] # 31C3 presentation with POC
```